unsup_vvs/network_training/models/simclr/resnet.py

```diff
@@ -128,7 +128,7 @@ def batch_norm_relu(inputs, is_training, relu=True, init_zero=False,
     inputs = tf.layers.batch_normalization(
         inputs=inputs,
         axis=axis,
-        momentum=FLAGS.batch_norm_decay,
+        momentum=FLAGS.batch_norm_decay if hasattr(FLAGS, 'batch_norm_decay') else .99,
```

Comment:
@chengxuz please confirm that this default value of `.99` is right

Reply:
yeah, I think it's fine. Will merge.